content/podcast/s03e06-brave/index.md

```diff
@@ -0,0 +1,64 @@
++++
+title = "Brave"
+date = 2025-01-07
+template = "episode.html"
+draft = false
+aliases = ["/p/s03e07"]
+[extra]
+guest = "Anton Lazarev"
+role = "Senior Software Engineer"
+season = "03"
+episode = "06"
```

Comment:
```suggestion
episode = "07"
```
The folder is also still numbered s03e06, but that was the community edition.

Reply:
Dang it. Thanks